### PR TITLE
Fix download url for rules_nodejs 0.32

### PR DESCRIPTION
**Pull request by meteorcloudy**
*Attention Googlers:* This repo has its Source of Truth in Piper. After sending a PR, you can follow http://g3doc/third_party/bazel_rules/rules_typescript/README.google.md#merging-changes to get your change merged.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: package.bzl [hunk 1]
```diff
@@ -31,7 +31,7 @@ def rules_typescript_dev_dependencies():
         http_archive,
         name = "build_bazel_rules_nodejs",
         sha256 = "06cb04f4f745e37d542ec6883a2896029715a591c0e44c5d250a268d3752f865",
-        urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.32.0/rules_nodejs-0.32.0.tar.gz"],
+        urls = ["https://github.com/bazelbuild/rules_nodejs/archive/0.32.0.tar.gz"],
```
Comment (Toxicable) [inline package.bzl]: This url is not correct, there is a difference between the released package and the archived one

Comment (meteorcloudy) [reply package.bzl]: Thanks, done.

**Comment by kyliau** [comment]
@meteorcloudy Could you please squash the commits?

**Comment by kyliau** [comment]
@meteorcloudy I squashed the commits and force pushed to your branch.
Created cl/255442293 to sync into google3.

**Pull request closed without merging**